recipes/xxhash/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class XxHash(ConanFile):
+    name = "xxhash"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Cyan4973/xxHash"
+    description = "Extremely fast non-cryptographic hash algorithm"
+    topics = ("conan", "hash", "algorithm", "fast", "checksum", "hash-functions")
+    license = "BSD-2-Clause"
```

Comment:
https://github.com/Cyan4973/xxHash/blob/a9054f397d7f41bc505638df3853b270eb9e7493/LICENSE#L30

```suggestion
    license = "BSD-2-Clause", "GPL-2.0-or-later"
```

Reply:
That's only for the cli, when `options.utility=True`.
But you're correct. Best to mention both.